Build.mak

```diff
@@ -0,0 +1,16 @@
+# Set of flags required to compile vibe.d without dub
+override DFLAGS += -version=VibeLibeventDriver -version=Have_openssl \
+	-I./submodules/vibed/source -I./submodules/libevent -I./submodules/openssl
+
+# Compile all vibe.d as a single static library because rdmd can't track
+# all imported dependencies correctly in this case
+$O/libvibed.a: $(shell find $C/submodules/vibed/source -type f -name '*.d')
```

Comment:
you can use `$(call find_files,.d,,$C/submodules/vibed/source)`, although probably not a huge gain if you don't want to also get the provided conversions (.d -> .o)